dotcom-rendering/src/client/userFeatures/user-features.ts

```diff
@@ -86,12 +93,18 @@ const persistResponse = (userBenefitsResponse: UserBenefits) => {
 	} else if (adFreeDataIsPresent() && !forcedAdFreeMode) {
 		removeAdFreeCookie();
 	}
+	if (userBenefitsResponse.allowRejectAll) {
+		setAllowRejectAllCookie(2);
```

Comment:
why 2 days?

Reply:
I'm not sure really, I guess to give a bit of wiggle room past the 24hr refresh period? This is the same as the ad-free cookie which was already in place so I used the same value.

I'm going to do a bit of an investigation into the expiry times we set on these cookies and when we explicitly delete them, so this may change in a future PR